### PR TITLE
Less aws requests

### PR DESCRIPTION
**Pull request by evansiroky**
### Checklist

- [x] Appropriate branch selected _(all PRs must first be merged to `dev` before they can be merged to `master`)_
- [x] Any modified or new methods or classes have helpful JavaDoc and code is thoroughly commented
- [x] The description lists all applicable issues this PR seeks to resolve
- [x] The description lists any configuration setting(s) that differ from the default settings
- [x] All tests and CI builds passing

### Description

This PR is built on top of #341.

This PR does a few things to reduce the amount of AWS requests that are performed as follows:

- Don't include ec2 instance data with OtpServers in API calls to get a Project or Projects
- Don't include ec2 instance data with OtpServers in API calls to get all servers
- Add new endpoint for getting a single OtpServer
- During MonitorServerStatusJobs only check instance health after waiting for 4 seconds instead of both before and after

This PR has a companion PR for the UI here: https://github.com/ibi-group/datatools-ui/pull/615

### CODE REVIEW

File: src/main/java/com/conveyal/datatools/manager/models/Project.java [hunk 1]
```diff
@@ -129,4 +130,17 @@ public void delete() {
...
     }
+
+    /**
+     * A MixIn to be applied to this project, for returning a single project, so that the list of otpServers is
+     * included in the JSON response.
+     *
+     * Usually a mixin would be used on an external class, but since we are changing one thing about a single class, it
+     * seemed unnecessary to define a new view.
+     */
+    public abstract static class ProjectWithoutOtpServers {
```
Comment (landonreed) [inline src/main/java/com/conveyal/datatools/manager/models/Project.java:141]: I don't understand why this is needed. Isn't `otpServers` omitted by default? The Mixin above is only needed to add the `otpServers` property.

File: src/main/java/com/conveyal/datatools/manager/models/OtpServer.java [hunk 2]
```diff
@@ -434,4 +434,17 @@ public void verifyS3WritePermissions() throws IOException, CheckedAWSException {
         client.putObject(s3Bucket, key, File.createTempFile("test", ".zip"));
         client.deleteObject(s3Bucket, key);
     }
+
+    /**
+     * A MixIn to be applied to this OtpServer, for returning a single OtpServer, without EC2InstanceSummaries that
+     * don't need to be written in JSON output.
```
Comment (landonreed) [inline src/main/java/com/conveyal/datatools/manager/models/OtpServer.java:440]: This comment is inverted.

**Review by landonreed — CHANGES_REQUESTED** [review]
Comments need some cleanup and I'm not sure that ProjectWithoutOtpServers needs to exist.

File: src/main/java/com/conveyal/datatools/manager/models/Project.java (hunk 1 above)
Comment (evansiroky) [reply src/main/java/com/conveyal/datatools/manager/models/Project.java:141]: Ah, you're right. Not sure why I added it.

**Comment by evansiroky** [comment]
I made the requested changes.

**Review by binh-dam-ibigroup — APPROVED** [review]
The incremental changes from #341  to #343 look fine to me.

**Review by landonreed — APPROVED** [review]

**Pull request merged**